ftw/tabbedview/statestorage.py

```diff
@@ -27,27 +28,29 @@ def __init__(self, context, tabview, request):
         self.context = context
         self.tabview = tabview
         self.request = request
+        self.member = api.get_current()
 
     def get_key(self):
+        if not self.member or self.member == AccessControl.getSecurityManager().getUser():
```

Comment:
I believe that the condition `self.member == AccessControl.getSecurityManager().getUser()` is wrong, it should be "is not logged in" or "is anonymous". I think I got that wrong in 66f3887b36ba62b3c990e954f97c33ec23daf225.

The idea of this condition is that we need to awoid having a statestorage key for anonymous users, since the state is a personal thing and should not be shared among anonymous users.

I therefore propose to change this condition to:
```python
def get_key(self):
    if api.user.is_anonymous():
        return None
```